### DISCUSSION TITLE
Which three countries does Mexico share a land border with?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#96] [Galaxy Trivia] [answered]
**Hint:** US, BZ, GT

**Comment by FSa-git** [comment, marked as answer]
USA, Guatemala and Belice!

**Reply by ParanoidUser** [reply]
Congrats, you got `Galaxy Brain` badge! 🎉

What's next?
- Try to earn https://github.com/ParanoidUser/yolo/discussions/18
- Give ⭐ to the repo 🤩
- Scribe your name & new badge into [history](https://github.com/ParanoidUser/yolo/blob/main/docs/CHANGELOG.md)
- Post your own questions to the community 😉
